README.md

```diff
@@ -640,6 +640,23 @@ Run the Assembly Workflow to fix any merge conflict in ASSEMBLY_REPORT.md and an
 
 Restart your Worker after merging `main` into your branch and before running Assembly again.
 
+## Autogenerate table of contents
```

Comment:
We should probably create a README specifically for folks working on Assembly - and that would be a good place for this.